tests/system/action/motion_category/test_delete.py

```diff
@@ -0,0 +1,22 @@
+from openslides_backend.shared.exceptions import DatabaseException
+from tests.system.base import BaseSystemTestCase
+from tests.util import get_fqid
+
+
+class MotionCategorySystemTest(BaseSystemTestCase):
+    def test_delete_correct(self) -> None:
+        self.create_model(get_fqid("motion_category/111"), {"name": "name_srtgb123"})
+        response = self.client.post(
+            "/", json=[{"action": "motion_category.delete", "data": [{"id": 111}]}],
+        )
+        self.assertEqual(response.status_code, 200)
+        with self.assertRaises(DatabaseException):
+            self.datastore.get(get_fqid("motion_category/111"))
```

Comment:
You can use the parameter `get_deleted_model` to also get deleted models. Maybe that's the better approach here. Every model has a field `meta_deleted` which is set to true on delete, you can check that here. (The DatabaseException could come from everything...)

Reply:
Updated the delete test to use the get_deleted_mode feature.